### ISSUE TITLE
How to get current path

### ISSUE THREAD

**Issue by theweiweiway** [lulupointu/vrouter#8]
Hey Lulu, is there a way to get the current path? I see:
```
VRouteElementData.of(context).pathParameters
VRouteElementData.of(context).name
VRouteElementData.of(context).vChildName
VRouteData.of(context).pathParameters
VRouteData.of(context).queryParameters
```

but none of those seem to be giving me the current path.

Thanks!

**Comment by lulupointu** [comment]
You can have it using `VRouterData.of(context).url`

**Comment by theweiweiway** [comment]
Amazing, thanks!

**Comment by rohitranjan1991** [comment]
How do I get the current path name, not the url?

**Comment by lulupointu** [comment]
Update to `1.1.4+5` and you can now use `VRouter.of(context).path` :+1:

**Comment by rohitranjan1991** [comment]
Thanks 👍

**Comment by rohitranjan1991** [comment]
@lulupointu, any update on this?

**Comment by lulupointu** [comment]
Sorry @rohitranjan1991, it's hard to follow when the issue is closed. I moved the issue to https://github.com/lulupointu/vrouter/issues/88 since it's not the same issue as this. Please follow up there 😊
